includes/admin/reporting/reports.php

```diff
@@ -486,6 +486,7 @@ function edd_register_downloads_report( $reports ) {
 		$dates         = Reports\get_filter_value( 'dates' );
 		$exclude_taxes = Reports\get_taxes_excluded_filter();
 		$label         = $options[ $dates['range'] ];
+		$filter        = Reports\get_filters();
```

Comment:
We should remove `$filter` from the `use()` statements instead since it's not being used inside of the callbacks.

Reply:
I'll go ahead and verify that in all the registered ones. Will update this shortly.

Reply:
All set @spencerfinnell, let me know if this is better.